apps/core-app/src/main/modules/network/network-service.ts

```
@@ -28,8 +25,8 @@ import {
   toTfileUrl
 } from '@talex-touch/utils/network'
 import { app, session } from 'electron'
-import { APP_FOLDER_NAME } from '../../config/default'
 import { getMainConfig, saveMainConfig } from '../storage'
+import { resolveRuntimeRootPath } from '../../utils/app-root-path'
```

ESLint

[error] Expected "../../utils/app-root-path" to come before "../storage".

(perfectionist/sort-imports)